quarkus.ls/com.redhat.quarkus.ls/src/test/java/com/redhat/quarkus/services/ApplicationPropertiesCompletionTest.java

```diff
@@ -140,4 +140,18 @@ public void completionAfterProfil() throws BadLocationException {
 		testCompletionFor(value, true, c("quarkus.http.cors", "%dev.quarkus.http.cors=${1|false,true|}", r(0, 0, 5)));
 	}
 
+	@Test
+	public void noCompletionForExistingProperties() throws BadLocationException {
+		String value = "quarkus.http.cors=false\r\n" + //
+				"|";
+		testCompletionFor(value, false);

```

Comment:
This test doesn't check the completion for existing completion. I suggest you that you use 

```
QuarkusAsset#testCompletionFor(String value, boolean snippetSupport, String fileURI, Integer expectedCount,
			QuarkusProjectInfo projectInfo, CompletionItem... expectedItems)
```

you create your own QuarkusProjectInfo  with 2 properties (ex: quarkus.application.name and quarkus.application.version)
```

In the same test you test 

 * with value="|" you should have 2 properties (by setting expectedCount=2)
 * with value="quarkus.application.name=\n|" you should have 2 properties (by setting expectedCount=1)

Reply:
Oops, I should have been more attentive. Thanks for the correction.